### ISSUE TITLE
The accuracy of the validation set is 0，and the loss is always around 13 

### ISSUE THREAD

**Issue by ConnyLamp** [zihangJiang/TokenLabeling#17]
Hello! I use ILSVRC2012_img_train and ILSVRC2012_img_val, and use the provided label_top5_train_nfnet from Google Drive. I train lv-vit-s with batch_size 64 without apex for one epoch. Thanks for your advice.

**Comment by zihangJiang** [comment]
Hi @ConnyLamp, can you try to rename `ILSVRC2012_img_train` & `ILSVRC2012_img_val` as `train` & `val` and organize the data folder as stated here https://github.com/zihangJiang/TokenLabeling#requirements.

**Comment by ConnyLamp** [comment]
> Hi @ConnyLamp, can you try to rename `ILSVRC2012_img_train` & `ILSVRC2012_img_val` as `train` & `val` and organize the data folder as stated here https://github.com/zihangJiang/TokenLabeling#requirements.

是的，我使用了你们提供的脚本来组织文件结构，但问题仍然存在。我使用的数据集是imagenet2012，是不是要在这个数据集上生成token-label-data，不能直接使用你们提供的token-label-data呢？

**Comment by ConnyLamp** [comment]
> Hi @ConnyLamp, can you try to rename `ILSVRC2012_img_train` & `ILSVRC2012_img_val` as `train` & `val` and organize the data folder as stated here https://github.com/zihangJiang/TokenLabeling#requirements.

是的，我使用了你们提供的脚本来组织文件结构，但问题仍然存在。我使用的数据集是imagenet2012，是不是要在这个数据集上生成token-label-data，不能直接使用你们提供的token-label-data呢？

**Comment by zihangJiang** [comment]
Can you list your command line for training and the log in `summary.csv`?

**Comment by ConnyLamp** [comment]
> Can you list your command line for training and the log in `summary.csv`?

bash ./distributed_train.sh 4 ./datasets --model lvvit_s -b 64 --apex-amp --img-size 224 --drop-path 0.1 --token-label --token-label-data ./datasets/label_top5_train_nfnet --token-label-size 14 --model-ema
[summary.csv](https://github.com/zihangJiang/TokenLabeling/files/7144494/summary.csv)

**Comment by zihangJiang** [comment]
Our token label data is also generated on ImageNet dataset. 
The batch size (4x64) you use is too small for the default learning rate. You need to change the learning rate to 4.e-4 by using `--lr 4.e-4`. And since the model ema is enabled, you may need to wait like 10 epochs. 
If you are still experiencing the issue, you can first try with the [timm](https://github.com/rwightman/pytorch-image-models) repo to train without token labeling to make sure your dataset is correctly organized.

Attaching our log for LV-ViT-S for your reference
[summary.csv](https://github.com/zihangJiang/TokenLabeling/files/7145062/summary.csv)

**Comment by ConnyLamp** [comment]
> Our token label data is also generated on ImageNet dataset.
> The batch size (4x64) you use is too small for the default learning rate. You need to change the learning rate to 4.e-4 by using `--lr 4.e-4`. And since the model ema is enabled, you may need to wait like 10 epochs.
> If you are still experiencing the issue, you can first try with the [timm](https://github.com/rwightman/pytorch-image-models) repo to train without token labeling to make sure your dataset is correctly organized.
> 
> Attaching our log for LV-ViT-S for your reference
> [summary.csv](https://github.com/zihangJiang/TokenLabeling/files/7145062/summary.csv)

谢谢您的解答，问题解决了~

**Comment by zihangJiang** [comment]
Great! I'll close this.
Feel free to reopen this issue if you have any other further questions.
